src/Storage/Device/S3.php

```diff
@@ -255,6 +259,24 @@ public function delete(string $path, bool $recursive = false): bool
         return true;
     }
 
+    /**
+     * Delete file in given path, Return true on success and false on failure.
+     *
+     * @see http://php.net/manual/en/function.filesize.php
+     *
+     * @param string $path
+     *
+     * @return bool
```

Comment:
I dont think we need the duplicate PHPDocs if they already exist on the parent's class.

Reply:
But we have it in all other methods

Reply:
Lets leave it then, its a duplicate but nothing serious.